cfgov/cfgov/settings/base.py

```diff
@@ -502,7 +505,7 @@
     'universal.iperceptions.com')
 
 # These specify where we allow fonts to come from
-CSP_FONT_SRC = ("'self'", 'fast.fonts.net')
+CSP_FONT_SRC = ("'self'",'data:', 'fast.fonts.net', 'fonts.google.com')
```

Comment:
Trivial: missing space between the first two items in this list.

Reply:
Hmm... Is the first one double-quoted because the string value has to include the literal single quote characters? `'self'`?

If so, would it be better for consistency to double-quote the rest of the values in this list? Does flake8 care about this at all?

Reply:
Correct on the literal quotes being neccessary for self. I've double-quoted the whole list now

Reply:
> Does flake8 care about this at all? 

We don't lint the settings files currently... for reasons which I think might be obvious ;)

Reply:
Hah!